rlberry/agents/agent.py

```diff
@@ -20,6 +20,10 @@
 class Agent(ABC):
     """Basic interface for agents.
 
+    .. note::
+
+        Classes that implement this interface should send ``**kwargs`` to :code:`Agent.__init__()`
+
```

Comment:
proposition : instead of this, put at the end of the Parameters the lines 
```
...
    Parameters
    ----------
    ...
    _default_writer_kwargs : dict (optional)
        Parameters to initialize DefaultWriter (attribute self.writer).
    kwargs : key, value mappings
        keyword arguments passed down to :class:`~rlberry.agents.Agent`.__init__.
```
This will give a link to the Agent API page in the doc

Reply:
I think it would be weird to have it in the parameters section, since the first thing we do in `Agent` is an assertion

```python
 assert kwargs == {}, "Unknown parameters sent to agent:" + str(kwargs.keys())
```

The ``**kwargs`` are here just to ensure that `AgentManager` can send parameters to `Agent` that are not handled by the user (such as output_dir, thread/process id etc.), in case we add more parameters of this kind to the `Agent` constructor.

Reply:
Ok. But at the same time, from a user perspective this is a parameter and the user see it in the parameters of init, he will search for it in the parameter section. You suppose that the user will read your note before going to the parameters...which is not certain I think. 

For reference this is also how it is done in [seaborn](https://seaborn.pydata.org/generated/seaborn.lineplot.html)